pkg/api/resource/helpers_test.go

```diff
@@ -60,3 +63,252 @@ func TestDefaultResourceHelpers(t *testing.T) {
 		t.Errorf("expected %v, actual %v", resource.BinarySI, resourceList.Memory().Format)
 	}
 }
+
+func TestPodRequestsAndLimits(t *testing.T) {
+	tests := []struct {
+		name         string
+		pod          *api.Pod
+		expectreqs   map[api.ResourceName]resource.Quantity
+		expectlimits map[api.ResourceName]resource.Quantity
+	}{
+		{
+			name: "InitContainers resource < Containers resource",
+			pod: &api.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					UID: uuid.NewUUID(),
+				},
+				Spec: api.PodSpec{
+					InitContainers: []api.Container{
+						{
```

Comment:
This could benefit from a small helper func that returned this struct.  E.g. 

```
func makeContainer(cpuReq, cpuLim, memReq, memLim string) api.Container {
    // ....
}
```

and used as:

```
InitContainers: []api.Container{
    makeContainer("100m", "200m", "100Mi", "200Mi")
}
```

Reply:
Done, Thanks for your review